src/lint/lint.ts

```diff
@@ -45,10 +45,9 @@ export async function lint(options: Options, config: ProjectConfig) {
   const rules = lintLib.registry.getRules(ruleCodes || lintOptions.rules);
   const filter = new WarningFilter({
     warningCodesToIgnore: new Set(lintOptions.ignoreWarnings || []),
-    minimumSeverity: Severity.WARNING
+    minimumSeverity: options.minSeverity || lintOptions.minSeverity || Severity.WARNING,
```

Comment:
This doesn't compile because project config doesn't have minSeverity. Options:

* don't include the reference
* reference it using a string literal `lintOptions['minSeverity']`

In either case we should have a TODO here to add it to project config